tsdb/index/tsi1/file_set.go

```diff
@@ -88,7 +81,7 @@ func (fs *FileSet) Size() int64 {
 	for _, f := range fs.files {
 		total += f.Size()
 	}
-	return total + int64(fs.manifestSize)
```

Comment:
I'm not sure about this one (the fact that `manifestSize` is always `0` could be a bug). Will check internally with teammates who know this code better